crawl-ref/source/dat/descript/monsters.txt

```diff
@@ -1405,6 +1405,12 @@ howler monkey
 An aggressively territorial primate, known for the astonishing variety and
 volume of noise it can produce.
 %%%%
+hulking thunderguard
```

Comment:
Does this not have a species? maybe make it an ironclad or a palentonga or something

Reply:
I see its a human below, why not name it similar to other vaults monsters. Ironheart?

Reply:
Sure, could do. No strong opinion.